staking_deposit/intl/en/cli/generate_keys.json

```diff
@@ -19,7 +19,7 @@
         },
         "arg_execution_address": {
             "help": "The 20-byte (Eth1) execution address that will be used in withdrawal",
-            "prompt": "Please enter the 20-byte execution address for the new withdrawal credentials. Note that you CANNOT change it once you have set it on chain.",
+            "prompt": "Please enter the optional 20-byte execution address for the new withdrawal credentials. Note that you CANNOT change it once you have set it on chain.",
```

Comment:
I would add something like this at the end of this message:

> Please enter the optional 20-byte execution address for the new withdrawal credentials. Note that you CANNOT change it once you have set it on chain. **Use a wallet address that you will have full control over for foreseeable future.**

What do you think?